Code/Source/Entity/MeshEntityManager.h

```diff
@@ -26,14 +26,15 @@ namespace RGL
     {
     public:
         explicit MeshEntityManager(AZ::EntityId entityId);
+
         MeshEntityManager(const MeshEntityManager& other) = delete;
         MeshEntityManager(MeshEntityManager&& other) = delete;
         MeshEntityManager& operator=(MeshEntityManager&& rhs) = delete;
         MeshEntityManager& operator=(const MeshEntityManager&) = delete;
         ~MeshEntityManager() override;
 
     protected:
-        // AZ::Render::MeshComponentNotificationBus overrides
+
```

Comment:
Why this line was removed? It is correct to have base class hint here

Reply:
By mistake